### DISCUSSION TITLE
Reset to default renderFunction

### DISCUSSION THREAD

**Discussion by rodsmyname** [katspaugh/wavesurfer.js#3922] [Q&A]
I want to be able to toggle between a custom render function and the default one. Is there a way to do that using ```setOptions```?

If I don't include renderFunction in the setOptions it redraws it with without changing it. So if I initialize it with a custom rendered wave, it will not change from that.

**Comment by katspaugh** [comment]
Try passing renderFunction: null. I’m afk so cannot check but it might work.
